components/CookieBanner/index.js

```diff
@@ -51,17 +51,18 @@ const CookieBanner = ({ cookies }) => {
   });
 
   useEffect(() => {
+    const cookies = cookie.parse(document.cookie ?? {});
```

Comment:
Not passing cookies as a prop to this component makes it harder to unit test, as instead of just passing the data to it we need to mock out `document`. Although I think this is sufficiently covered by Cypress tests anyway - so probably not a huge concern.